README.md

```diff
@@ -49,22 +49,59 @@ Here we list formats that have been used or proposed thus far in the community (
 
 ## Standard Matrices
 
-We benchmark data formats and APIs on standard matrices. These matrices are found at the following S3 bucket.
-
-[TODO add S3 bucket details]
-
-[TODO describe matrices below]
+We benchmark data formats and APIs on standard matrices. These matrices are found in the public S3 bucket `matrix-format-test-data`.
+
+That bucket has two relevant folders, `matrices` and `sources`. The `sources` folder contains raw data from a number of different data sources, in the form that they were originally delivered. These are meant to span a range of matrix sizes and properties so we can develop tests for diverse use cases:
+
+| Source Name | Description |
+| ----------- | ----------- |
+| tenx_mouse_neuron_1M | The 1.3 million mouse neuron dataset released by 10x. |
+| tenx_mouse_neuron_20k | A sample of 20k cells from the 1.3 million cell dataset, also provided by 10x. |
+| immune_cell_census_cord_blood | Cord blood cells from the Immune Cell Atlas dataset released for the HCA preview. 384,000 cells sequenced with 10x.|
+| immune_cell_census_bone_marrow | Bone marrow cells from the Immune Cell Atlas dataset released for the HCA preview. 378,000 cells sequenced with 10x. |
+| GSE84465_whole | 3589 cells sequenced with SmartSeq2 from GEO Series GSE84465. |
```

Comment:
link to the GEO page:
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84465

Also if other datasets can be linked to a relevant page, we better add the links here.

Reply:
Yeah, good idea. I'll add links.